src/map/entities/charentity.cpp

```diff
@@ -492,6 +492,7 @@ void CCharEntity::ClearTrusts()
     for (auto trust : PTrusts)
     {
         trust->PAI->Despawn();
+        loc.zone->DeletePET(trust);
```

Comment:
sorry i misremembered what this function does, and this line should be removed as it doesn't do what i thought it did
i had a look a while after and saw that instead of freeing the pet from memory it will only remove it from the zone's pet list, leaving it hanging around in memory